### ISSUE TITLE
Feature: Invite My Team

### ISSUE THREAD

**Issue by zacharysierakowski** [ox-eng/ox#1]
## tl;dr

We need to make a thoughtful UI that makes it crazy easy and simple to invite other team members to Ox, and assign them to the proper team (that should already have the proper permissions)

## Audiences

- "Champion" ie the first person in a company to use Ox, prove the concept for themselves first, then invite their team.
- Team member of a company that purchased Ox
- Manager of a team that purchased Ox

## Requirements

- Like the above, Ox's invite my team UI should require very few clicks and be delightful and very clear. This is a main CTA that we want to draw the user's attention to and ensure they complete
- In the invite process, the manager should be able to quickly designate:
  - What team the invited user should be on with a dropdown menu of their configured teams
  - What role the user should have from existing roles with a dropdown menu
    - current roles are:
      - manager
      - team member
- This UI should check to see if the email of the invitee has the same domain as the invitor. If the email domain is different, the language should switch to "looks like this person isn't a part of your team. Would you like to refer another team to Ox" and the options for teams and role should disappear to reduce cognitive load of referring someone new.

**Comment by zacharysierakowski** [comment]
Closing this to keep everything in clickup!
